src/components/Menu.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <!-- Application Navigation || Menubar -->
```

Comment:
I guess their is no need of this comments

Reply:
removed